padl/dumptools/var2mod.py

```diff
@@ -597,28 +605,52 @@ def find_globals(node: ast.AST, filter_builtins: bool = True) -> Set[Tuple[str,
     return globals_
 
 
+def _check_and_make_increment(value: ScopedName, targets: Set):
+    """Apply Increment on *value* ScopedName if necessary.
+
+    Increment means the increment of *value.n*.
+    *value.n* signifies which one of the usages of a *name* in the code does this
+     *scoped_name* points to.
+    0 being the most recent, and increase integer representing the older usages.
+
+    :param value: Target ScopedName for increment
+    :param targets: Set of ScopedName for increment
```

Comment:
I would consider rephrasing the docstrings to something like 
```python
:param value: ScopedName to be incremented (if necessary)
:param targets: Set of ScopedName's to compare to *value*
```
Right now it's a little confusing that `value` is described as the `Target ScopedName` when we also have a parameter called `targets`.

Reply:
thanks updated it.